hadoop-tools/hadoop-aws/pom.xml

```
@@ -312,6 +316,69 @@
       </properties>
     </profile>
 
+    <!-- Adds a *test only* java 17 build profile-->
+    <profile>
+      <id>java-17-or-later</id>
```

```suggestion
      <id>jdk17+</id>
```
better to follow
https://github.com/apache/maven-apache-parent/blob/ce14a4641f0f21f98e51f564e9d290910ae75e7c/pom.xml#L500